content/en/docs/protocol/whitepaper.md

```diff
@@ -19,9 +19,9 @@ description: >
 
 The Aqua Protocol (AQP) is a data accountability and exchange protocol between
 hosts in peer-to-peer environments. The AQP is used to realize the goal of
-[accounting for data origin and history](Data_Accounting.md). The description
+[accounting for data origin and history](data-accounting.md). The description
 for the proof-of-concept implementation of the AQP can be found in [this
-page](Reference_Implementation_Architecture.md).
+page](../implementations/reference-architecture.md).
```

Comment:
It works on GH, but there is no way to test if this also works on aqua-protocol.org. I will just merge this PR now and see what it looks like.